src/modules/Lan.rb

```diff
@@ -659,6 +659,10 @@ def Import(settings)
       DNS.Import(settings["dns"] || {})
       Routing.Import(settings["routing"] || {})
 
+      # Ensure that the /etc/hosts has been read to no blank out it in case of
+      # not defined <host> section (bsc#1058396)
+      Host.Read
```

Comment:
hm. Now when ordering is guaranteed, you can change it to

```
Host.Read if !Host.initialized
```

Reply:
ah bad idea ```Host`` already checks that, so no need to do it here.

Reply:
Exactly, if it is already initialized then it just returns.